pkgs/development/compilers/blueprint/default.nix

```diff
@@ -36,9 +44,31 @@ stdenv.mkDerivation (finalAttrs: {
     gobject-introspection
   ];
 
+  env = {
+    # Fontconfig error: Cannot load default config file: No such file: (null)
+    FONTCONFIG_FILE = makeFontsConf { fontDirectories = [ ]; };
+  };
+
+  # Fontconfig error: No writable cache directories
```

Comment:
Please move the comment next to the export so that it does not get detached when more stuff is added.